app/serializers/foodbags_index_serializer_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class FoodbagsIndexSerializerSerializer < ActiveModel::Serializer
+    attributes :id, :pickuptime, :status
+    attribute :created_at do
+    object.created_at.to_formatted_s(:db)
```

Comment:
what is this? just to understand :)

Reply:
This is to format the date to yyyy-mm-dd hh:mm:ss